utils/controleval.py

```diff
@@ -138,6 +219,11 @@ def create_parser():
         required=True)
     statsparser.add_argument(
         "-p", "--product", help="product to check has required references")
+    statsparser.add_argument(
+        "-o",
+        "--output_json_file",
+        type=argparse.FileType('w'),
+        help="json file output")
```

Comment:
By doing this for JSON and not for the non-JSON path, are we setting the precedence that a user should be able to write any output from this tool to a file?

Reply:
that's a good point, how does it look now?

Reply:
Today - I don't think this script supports writing to a file natively (but you can surely redirect it as the caller)

Reply:
Sorry, it might be a stupid question, could you elaborate on that?

Reply:
Not at all - I should have been more explicit in my initial comment.

What I meant here was to leave out the support for writing to a file and print the JSON output to stdout, just like how we print the report to today.

That would allow someone to redirect the output to a file if they want to persist it:

```
$ controleval.py ... -o json | report.json
```

Or, a caller can pipe it into some other useful utility, like `jq` if they want to query specific details of the summary:

```
$ controleval.py ... -o json | jq .
```

Reply:
thanks for the explanation, that makes sense!